Scripts/build-native-mac.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+export sentryNativeRoot=$1
+export sentryArtifactsDestination=$2
+
+rm -rf "${sentryArtifactsDestination}/bin/"*
+rm -rf "${sentryArtifactsDestination}/include/"*
+
+cmake -S "${sentryNativeRoot}" -B "${sentryNativeRoot}/build" -D SENTRY_BACKEND=crashpad -D SENTRY_SDK_NAME=sentry.native.unreal -DCMAKE_OSX_ARCHITECTURES="arm64;x86_64" -DCMAKE_OSX_DEPLOYMENT_TARGET=10.14
```

Comment:
you may need to specify PYTHON_EXECUTABLE here (just pointing out, if it comes up in CI) - see https://github.com/getsentry/sentry-unity/pull/699/files#diff-50e91c82311ea26f2a73202525dfdf2b0a89c178ee666b2bf3ad4c84ac4c06dcR216